teps/0084-endtoend-provenance-collection.md

```diff
@@ -112,21 +113,21 @@ With "chains", we are able to capture the signed provenance for individual `task
 * Attestation records are captured in popular formats like "in-toto"
 * These record(s) are automatically signed  and pushed to different storage backends (same techniques that chains employed)
 * The attestation record(s) for pipeline execution are self-contained to perform independent provenance audit
-* The attestation process is transparent to the user, in the sense user do not need to change their pipeline.
+* The attestation process is transparent to the user, in the sense user do not need to change their pipeline. However, changes to the pipeline and/or task definition may be required to comply with expected format
```

Comment:
thanks @lcarva I am trying to understand this.

Does this say that the task and pipeline specifications will be updated? Or the definitions/CRDs available in cluster will be updated? 

Also, what kind of changes are we foreseeing?

Apologies if you have addressed this in the PoC, I haven't review it yet.

Reply:
I'm trying to capture here that in order for Chains to sign/attest a PipeilneRun, the PipelineRun (and in turn the Pipeline definition) must conform to what Chains expects. For example, naming a pipeline result as IMAGE_URL is essential for type hinting. This is also true today when generating TaskRun attestations. I didn't want to give the impression that Chains would just work with whichever Pipeline definitions a user may have.